### PR TITLE
Add French translation

### PR DESCRIPTION
**Pull request by defvs**
Fully translated string.xml to french

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -157,7 +157,7 @@
     <string name="preference_load_message_history_key" translatable="false">load_message_history_key</string>
     <string name="preference_message_history_header">Message history</string>
     <string name="preference_message_history_dashboard_title">Open dashboard</string>
-    <string name="preference_message_history_dashboard_key">message_history_dashboard_key</string>
+    <string name="preference_message_history_dashboard_key" translatable="false">message_history_dashboard_key</string>
     <string name="preference_message_history_dashboard_summary">Learn more about the service and disable message history for your own channel</string>
     <string name="preference_channel_data_header">Channel data</string>
     <string name="preference_developer_header" translatable="false">Developer options</string>
```
[the remark this answers is not in the document]
Comment (defvs) [reply app/src/main/res/values/strings.xml]: Fixed in a51132847bedd41a5916fe12d0cf8131931dba1c

**Comment by defvs** [comment]
Fixed line returns in cb4793a7b2c028d1cc7f60de1819456451e8603b
Translated plurals in a51132847bedd41a5916fe12d0cf8131931dba1c

**Comment by defvs** [comment]
Found some more strings not in resources: 
image

**Comment by flex3r** [comment]
> Also your layout here is not great:
> image

Yeah I'm gonna fix the layout by moving the options to a new line I think. Will do after this PR. 
Since your other PR is merged the only thing missing is the translation for the new setting

**Comment by defvs** [comment]
Ready to be merged.

**Pull request merged**